pkg/transform/kubernetes/kubernetes.go

```diff
@@ -41,7 +41,7 @@ func (t *Transformer) CreateServices(o *object.Service) ([]runtime.Object, error
 			s.Spec.Ports = append(s.Spec.Ports, api_v1.ServicePort{
 				Name:       fmt.Sprintf("port-%d", p.Port.ServicePort),
 				Port:       int32(p.Port.ServicePort),
-				TargetPort: intstr.FromInt(p.Port.HostPort),
+				TargetPort: intstr.FromInt(p.Port.ContainerPort),
```

Comment:
@pradeepto you should probably fetch a new master before the rebase https://github.com/redhat-developer/opencompose/commit/ce8564a997a8868c3d7f5ce6056f55a085090792